taskdart.dart

```diff
@@ -0,0 +1,148 @@
+/*Create a new Dart file (e.g., task_list.dart) to build the task list app.
+Implement a menu-driven interface that allows users to, view update, and delete tasks using the Task class.*/
+import 'dart:io';
+
+void main() {
+  //Task task1 = Task(1, 'Buy groceries', 'Buy milk, eggs, and bread', false);
+  //Task task2 = Task(2, 'Read a book', 'Read "The Catcher in the Rye"', true);
+
+  //print('Task 1: ${task1.title}, Completed: ${task1.isCompleted}');
+  //print('Task 2: ${task2.title}, Completed: ${task2.isCompleted}');
+  app taskListApp = app();
+  taskListApp.ap();
+}
+
+class Task {
+  // task class
+  int id;
+  String title;
+  String description;
+  bool isCompleted;
+
+  Task(
+      {required this.id,
+      required this.title,
+      required this.description,
+      required this.isCompleted});
+}
+
+class app {
```

Comment:
Class names should follow standard naming convensions, should start with a capital